package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "linkfree",
-  "version": "0.70.11",
+  "version": "0.70.1",
```

Comment:
You still need to make some changes! Also, remove the **empty endline** from `package-lock.json` file! 

```suggestion
  "version": "0.70.11",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Noted. What changes should I make?

Reply:
In order to solve the merge conflicts, your PR should not contain any changes in `package-lock.json` file 

Commit the changes I suggested above in line `3`, and remove the empty line from the end of the `package-lock.json` file!

Reply:
Thanks for the feedback. I understand the changes I'm supposed to make, however, I'm still confused about the version I'm required to use. Is it "version": "0.70.1"    or    "version": "0.70.11" ? Please elaborate so I can solve the merge conflicts.